tests/_helpers/github-workflows.js

```diff
@@ -0,0 +1,70 @@
+const GitHubClient = require( './github-client' );
+
+function verifyWorkflowStatus( workflowObject, reportCurrentStatus, waitingTime ) {
+	return new Promise( ( resolve, reject ) => {
+		if( workflowObject.status === 'completed' ) {
+			resolve( workflowObject );
+			return;
+		}
+		waitingTime = waitingTime || 1000;
+
+		reportCurrentStatus( workflowObject, waitingTime );
+
+		setTimeout( async () => {
+			const workflow = await getWorkflowRun( workflowObject.id );
+			// Give some time between rechecks
+			const result = await verifyWorkflowStatus( workflow.data, reportCurrentStatus, waitingTime + 3500 );
+			resolve( result );
+		}, waitingTime );
+	} );
+}
+
+async function getRunningActions() {
+	const result = await GitHubClient.request( 'GET', '/repos/{owner}/{repo}/actions/runs', {
+		headers: {
+			authorization: 'token ' + process.env.AUTH_KEY
+		},
+		owner: process.env.OWNER ,
+		repo: process.env.REPO
+	  });
+
+	return result;
+}
+
+async function getWorkflowRun( workflowId ) {
+	const result = await GitHubClient.request(
+		'GET',
+		'/repos/{owner}/{repo}/actions/runs/{run_id}',
```

Comment:
In other places like [https://github.com/ckeditor/ckeditor4-workflows-common/pull/15/files#diff-6f6766ef88da83cef9c00870cc460545c8ce3125106eb5dd135027274cf06b0cR23,](https://github.com/ckeditor/ckeditor4-workflows-common/pull/15/files#diff-6f6766ef88da83cef9c00870cc460545c8ce3125106eb5dd135027274cf06b0cR23,) you use single line for this, and here is split into multiple lines - it will be good to keep it consistent across entire code base.

Reply:
Seems to be a little too long. Inlined :)